rules/workspace_status/status_merger.tmpl.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# This script merges the two workspace status files into one.
+
+set -o errexit
+
+echo "$(cat "{info_file}") $(cat "{version_file}")" | "{jq}" --slurp add > "{workspace_status}"
```

Comment:
FWIW:

- `cat` takes multiple file names (it stands for con*cat*enate after all)
- `jq` takes files, so this is just
```suggestion
"{jq}" --slurp add "{info_file}" "{version_file}" > "{workspace_status}"
```